docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "3.9"
```

Comment:
A follow-up branch adds to this docker-compose also the containers needed for celery. I thought it was simplest to introduce that (along with integration tests for that) out from this branch that was already quite large.